tests/test_default_qubit.py

```diff
@@ -715,7 +748,7 @@ def test_args(self):
 
     
     @pytest.mark.parametrize("gate", set(qml.ops.cv.ops))
-    def test_unsupported_gate_error(self, qubit_device_2_wires, gate):
+    def test_unsupported_gate_error(self, qubit_device_3_wires, gate):
```

Comment:
Todo: Unrelated tests should not have to be updated like this when new code is added. This test should be marked for deprecation.